ui/app/controllers/servers/server.js

```diff
@@ -0,0 +1,14 @@
+import Controller from '@ember/controller';
+
+export default class ServersServerController extends Controller {
+  get server() {
+    return this.model;
+  }
+
+  get breadcrumb() {
+    return {
+      label: this.server.name,
+      args: ['servers.server', this.server.id],
+    };
+  }
+}
```

Comment:
Is there a criteria fro when to use a list vs. a single object? The other controllers seem to use a list even if there's only one item. Perhaps this is something that should be standardized

Reply:
There's no criteria of when to use a single object versus a list. We use a list when we're in a route that wants to show that it is part of a hierarchy that it's not actually a part of in `router.js`.

Generally, that's the only use case for using a list. Everything else should render to depict the route's `model`.

Reply:
Yeah... I see the inconsistencies. Gonna re-do all the commits and clean this up tomorrow.